custom_components/tuya_local/devices/arlec_apr005ha_air_purifier.yaml

```diff
@@ -0,0 +1,101 @@
+name: Air Purifier
+products:
+  - id: dfgsetrdnabqeimr
+    name: Arlec Air Purifier APR005HA
+primary_entity:
+  entity: fan
+  translation_only_key: fan_with_presets
+  dps:
+    - id: 1
+      name: switch
+      type: boolean
+    - id: 3
+      name: preset_mode
+      type: string
+      mapping:
+        - dps_val: Auto
+          value: smart
+        - dps_val: Manual
+          value: normal
+        - dps_val: Sleep
+          value: sleep
+    - id: 4
+      type: integer
+      name: speed
+      range:
+        min: 1
+        max: 3
+secondary_entities:
+  - entity: switch
+    translation_key: ionizer
+    category: config
+    dps:
+      - id: 6
+        type: boolean
+        name: switch
+  - entity: switch
+    translation_key: uv_sterilization
+    category: config
+    dps:
+      - id: 9
+        type: boolean
+        name: switch
+  - entity: lock
+    translation_key: child_lock
+    category: config
+    dps:
+      - id: 7
+        type: boolean
+        name: lock
+  - entity: select
+    translation_key: timer
+    category: config
+    dps:
+      - id: 18
+        type: string
+        name: option
+        mapping:
+          - dps_val: cancel
+            value: "Off"
+          - dps_val: 1h
+            value: 1 hour
+          - dps_val: 2h
+            value: 2 hours
+          - dps_val: 3h
+            value: 3 hours
+          - dps_val: 4h
+            value: 4 hours
+          - dps_val: 5h
+            value: 5 hours
+          - dps_val: 6h
+            value: 6 hours
+          - dps_val: 7h
+            value: 7 hours
+          - dps_val: 8h
+            value: 8 hours
+          - dps_val: 9h
+            value: 9 hours
+          - dps_val: 10h
+            value: 10 hours
+          - dps_val: 11h
+            value: 11 hours
+          - dps_val: 12h
+            value: 12 hours
+  - entity: sensor
+    name: Air quality
+    class: pm25
+    dps:
+      - id: 2
+        type: integer
+        name: sensor
+        unit: µg/m³
+  - entity: sensor
+    name: Total time
```

Comment:
Total what time?  Maybe "Running time" is better, or just leave the name off and use the translations for "duration" class. Or if it is the time left for the timer, `translation_key: time_remaining` may be more suitable.

Reply:
Yeah, running time, I'll take the names out and use the translations